docs/user-guide/liveness/README.md

```diff
@@ -66,11 +66,13 @@ The [http-liveness.yaml](http-liveness.yaml) demonstrates the HTTP check.
       httpGet:
         path: /healthz
         port: 8080
+        httpHeaders:
+          X-Custom-Header: [ Awesome ]
```

Comment:
I think the vast majority of cases will be one key, one value.  Our API (and the documented conventions) eschew maps where the value is more than a simple primitive.

This can be a list of (key, value) or if you reallllly want to the map-like semantic a list of { key, []value}.